### PR TITLE
refactoring: code refactor

### PR DESCRIPTION
**Pull request by laljikanjareeya**
It Make sure the below things:

1. Fix formatting
2. Remove Extra line
3. Naming Convension based on [C# Codding Style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)
4. Added Missing XML document
5. Removed Unused Members
6. Removed Unused Methods
7. Improve Linq Query
8. Remove Unused Import
9. Use Async Method in sample

### CODE REVIEW

File: Google.Cloud.EntityFrameworkCore.Spanner.Samples/Snippets/ReadOnlyTransactionSample.cs [hunk 1]
```diff
@@ -49,16 +49,14 @@ public static async Task Run(string connectionString)
 
         // Create a new database context and insert a singer with the given id. This singer will not be visible
         // to the read-only transaction.
-        using (var writeContext = new SpannerSampleDbContext(connectionString))
```
Comment (olavloite) [inline Google.Cloud.EntityFrameworkCore.Spanner.Samples/Snippets/ReadOnlyTransactionSample.cs:52]: This indented context was intentional, as it should not be used (and should not be accessible) after this block.

File: Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs [hunk 2]
```diff
@@ -157,7 +157,7 @@ private static async Task MaybeCreateInstanceOnEmulatorAsync(string projectId, s
                         DisplayName = "Sample Instance",
                         NodeCount = 1,
                     },
-                }).PollUntilCompletedAsync();
```
[the remark this answers is not in the document]
Comment (laljikanjareeya) [reply Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs]: Not sure but both are returning the `Operation<Instance, CreateInstanceMetadata>`.

[the remark this answers is not in the document]
Comment (laljikanjareeya) [reply Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs]: Done!!

File: Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs [hunk 3]
```diff
@@ -172,16 +172,16 @@ private static async Task MaybeCreateDatabaseOnEmulatorAsync(DatabaseName databa
...
 
             var instanceName = InstanceName.FromProjectInstance(databaseName.ProjectId, databaseName.InstanceId);
             try
             {
-                await databaseAdminClient.CreateDatabase(new CreateDatabaseRequest
+                await databaseAdminClient.CreateDatabaseAsync(new CreateDatabaseRequest
                 {
                     ParentAsInstanceName = instanceName,
                     CreateStatement = $"CREATE DATABASE `{databaseName.DatabaseId}`",
-                }).PollUntilCompletedAsync();
```
[the remark this answers is not in the document]
Comment (laljikanjareeya) [reply Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs]: Done!!

File: Google.Cloud.EntityFrameworkCore.Spanner.Samples/Snippets/ReadOnlyTransactionSample.cs (hunk 1 above)
Comment (laljikanjareeya) [reply Google.Cloud.EntityFrameworkCore.Spanner.Samples/Snippets/ReadOnlyTransactionSample.cs:52]: Done!!

File: Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (olavloite) [reply Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs]: > Not sure but both are returning the Operation<Instance, CreateInstanceMetadata>.

Yes, but the latter will not return until the long-running operation has actually finished executing. On the emulator this does not really make any difference, as the operations finish directly anyways, but if we were to run the same samples on real Spanner, we would risk getting `NotFound` errors as the instance / database creation would potentially not have finished yet when the sample is executed.

**Review by olavloite — APPROVED** [review]

**Pull request merged**